### ISSUE TITLE
Checkbox state doesn't change when pressing Space

### ISSUE THREAD

**Issue by TGlide** [aviflombaum/shadcn-rails#23]

**Comment by sergitejada** [comment]
Using windows 10 and firefox it works correctly.
Clicking on a check when pressing space works fine.

**Comment by TGlide** [comment]
> Using windows 10 and firefox it works correctly. Clicking on a check when pressing space works fine.

You are correct. The issues that appears is on the docs, when clicking the label, the checkbox changes. But if I then press space, it doesn't trigger another change, as expected. 

See the video attached for a comparison with the original shadcn ui.

https://github.com/aviflombaum/shadcn-rails/assets/26071571/97d9512e-4278-4d5a-8aad-c967a8048cdd
